src/api/common/utils/Utils.js

```diff
@@ -400,3 +400,14 @@ export function typedKeys<K: string, V>(obj: {[K]: V}): Array<K> {
 	return downcast(Object.keys(obj))
 }
 
+export function getAsNotLazy<T>(maybe: MaybeLazy<T>): T {
```

Comment:
I don't like the name with "not", maybe "unwrapMaybeLazy" or "getFromLazy"?

Reply:
im gonna go with "resolveMaybeLazy"

Reply:
Sounds better than unwrap, yeah